src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -60,7 +60,7 @@ public function __construct(float $origin = null, bool $morePrecision = false)
      *
      * @return self|null The child section or null when none found
      */
-    public function get($id)
+    public function get(?string $id)
```

Comment:
`string` (I don't see why we would need to allow `null` here)

Reply:
Because method `get()` is used in `open()` where you can pass `null` to create a new section.

See: 
- https://github.com/symfony/symfony/pull/32242/files#diff-bde580cd52bd6a217202b9e4708a0090R81 and 
- https://github.com/symfony/symfony/pull/32242/files#diff-3f476e6279b86e411dd4b005595483b7R66

Reply:
https://github.com/symfony/symfony/pull/32242/files#diff-3f476e6279b86e411dd4b005595483b7R66 only calls `get` when the id is not null. I would suggest to do the same in `open` (`if (null === $id || null === $session = $this->get($id)) {`). This way `get` does not need to be nullable because it just doesn't make sense.

Reply:
We we need to do the adjustment in 4.4 and also trigger a deprecation warning when calling `get(null)`

Reply:
> We we need to do the adjustment in 4.4 and also trigger a deprecation warning when calling get(null)

I opened a separate PR for the deprecation. https://github.com/symfony/symfony/pull/32408

Reply:
#32408 merged now (and 4.4 merged in master).